UltiSnips/tex.snippets

```diff
@@ -467,112 +387,4 @@ snippet ac "Acroynm normal" bi
 \ac{${1:acronym}}
 endsnippet
 
-snippet ni "Non-indented paragraph" bi
-\noindent
-endsnippet
-
-
-############
-# SECTIONS #
-############
-
-snippet chap "Chapter" wi
-\chapter{$1${VISUAL}}
-endsnippet
-
-snippet sec "Section" wi
-\section{$1${VISUAL}}
-endsnippet
-
-snippet sec* "Section*" wi
-\section*{$1${VISUAL}}
-endsnippet
-
-snippet sub "Subsection" wi
-\subsection{$1${VISUAL}}
-endsnippet
-
-snippet sub* "Subsection*" wi
-\subsection*{$1${VISUAL}}
-endsnippet
-
-snippet subsub "Subsection" wi
-\subsubsection{$1${VISUAL}}
-endsnippet
-
-snippet subsub* "Subsubsection" wi
-\subsubsection*{$1${VISUAL}}
-endsnippet
-
-snippet par "Paragraph" wi
-\paragraph{$1${VISUAL}}
-endsnippet
-
-snippet par* "Paragraph*" wi
-\paragraph*{$1${VISUAL}}
-endsnippet
-
-snippet subpar "Sub Paragraph" wi
-\subparagraph{$1${VISUAL}}
-endsnippet
-
-snippet subpar* "Sub Paragraph*" wi
-\subparagraph*{$1${VISUAL}}
-endsnippet
-
-snippet chapl "Chapter with label" wi
-\chapter{$1${VISUAL}}
-\label{cha:${2:${1/\W+/-/g}}}
```

Comment:
these are already defined in snipmate, so I removed them, but the labels here are more advanced, so we could leave these in as UltiSnips specific overrides.